lib/utils/resolveCliInput.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const _ = require('lodash');
-const minimist = require('minimist');
+const yargsParser = require('yargs-parser');
 
-const minimistOptions = {
+const yargsOptions = {
   boolean: ['help', 'version', 'verbose'],
   string: ['config'],
   alias: { config: 'c', help: 'h', version: 'v' },
```

Comment:
Let's add also `configuration: { "parse-numbers": false }`, and get rid of base64 workaround (it was in place as minimist coerced long (non-safe) number strings to number, and there was loss of information)